composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/monolog-bundle": "~2.1",
         "doctrine/common": "~2.4",
         "doctrine/doctrine-bundle": "~1.0",
-        "doctrine/phpcr-odm": "~1.1,>=1.1.0-beta2",
+        "doctrine/phpcr-odm": "~1.1,>=1.1.0-RC1",
```

Comment:
why not just lock to the 1.1.0 stable release as minimum version? i see no need in allowing beta versions.